cartoframes/viz/widgets/__init__.py

```diff
@@ -10,23 +10,18 @@
 from .formula_widget import formula_widget
 from .histogram_widget import histogram_widget
 from .time_series_widget import time_series_widget
-from ..widget import Widget
-from ..widget_list import WidgetList
-
-
-def _inspect(widget):
-    import inspect
-    lines = inspect.getsource(widget)
-    print(lines)
-
 
 __all__ = [
-    'Widget',
```

Comment:
Why are the main Widget classes being removed from the API?

Reply:
They're in the API but they were duplicated. They're not in `widgets` namespace (that's for built-in widgets)